### PR TITLE
Tcf 74 support upload files

### PR DESCRIPTION
**Pull request by flori950**
Header and map working again.
Language switcher works again. (Was a problem with bootstrap)
You can upload a csv or json and it will  be displayed in the block of notizen

### CODE REVIEW

File: frontend/src/app/shared/map/map.component.html [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<div id="map" style="height: 100%"></div>
```
Comment (thdrbrkw) [inline frontend/src/app/shared/map/map.component.html:1]: Why did you change that?
The map shouldn't have a fixed width or height initially.

**Comment by thdrbrkw** [comment]
I'm not sure yet, if we should be more restrictive with the files that can be uploaded. Also, we have to talk with BE if they rather want the file or the contents of the file

File: frontend/src/app/shared/map/map.component.html (hunk 1 above)
Comment (flori950) [reply frontend/src/app/shared/map/map.component.html:1]: Added new map component

**Comment by flori950** [comment]
added filter, edited the home icon, added pagenator, added errormessage some styling and translations

**Review by thdrbrkw — CHANGES_REQUESTED** [review]
image

The paginator does not work. 11 items are displayed when '10 items/page' are selected.
